lib/spree_virtual_gift_card/factories.rb

```diff
@@ -12,6 +12,11 @@
 
     factory :redeemable_virtual_gift_card do
       association :purchaser, factory: :user
+      inventory_unit do |gift_card|
+        gift_card.line_item.inventory_units.first ||
```

Comment:
This was an attempt at saying "if the line item has an inventory unit, then use one of those so all the associations are sane and reimbursements won't throw an exception" and if there is none, create one.